app/indexers/generic_work_resource_indexer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Generated via
+#  `rails generate hyrax:work_resource GenericWorkResource`
+class GenericWorkResourceIndexer < Hyrax::ValkyrieWorkIndexer
+  include Hyrax::Indexer(:basic_metadata)
+  include Hyrax::Indexer(:bulkrax_metadata)
+  include Hyrax::Indexer(:generic_work_resource)
+  include Hyrax::Indexer(:with_pdf_viewer)
+  include Hyrax::Indexer(:with_video_embed)
+
+  include HykuIndexing
+
+  # Uncomment this block if you want to add custom indexing behavior:
+  def to_solr
```

Comment:
Per your earlier comment, it seems like we need to add a generic_resource metadata yaml with admin not for generic work and image model. Would you agree @kirkkwang ?